CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright 2017 The Abseil Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+cmake_minimum_required(VERSION 2.8.12)
+project(absl)
+
+# config / options
+option(ABSL_ENABLE_PIC "Enable / disable position-independent code generation" OFF)
```

Comment:
Because CMAKE_POSITION_INDEPENDENT_CODE impact all targets in all cmake projects. If you include abseil as a sub_project. You might want that only on abseil itself.